### ISSUE TITLE
Unable to fix a trivial indentation issue

### ISSUE THREAD

**Issue by morozov** [slevomat/coding-standard#913]
```sh
$ composer show | grep -P 'coding-standard|php_codesniffer'

doctrine/coding-standard                       7.0.2   The Doctrine Coding Standard is a ...
slevomat/coding-standard                       6.1.5   Slevomat Coding Standard for PHP_C...
squizlabs/php_codesniffer                      3.5.4   PHP_CodeSniffer tokenizes PHP, Jav...
```

Consider the following code example:

```php
<?php

declare(strict_types=1);

$foo =
[
    'bar' =>
   [
    ],
];
```

The following violations are reported:
```
------------------------------------------------------------------------------------------------
FOUND 2 ERRORS AFFECTING 2 LINES
------------------------------------------------------------------------------------------------
 8 | ERROR | [x] Empty array declaration must have no space between the parentheses
   |       |     (Squiz.Arrays.ArrayDeclaration.SpaceInEmptyArray)
 9 | ERROR | [x] Array close brace not indented correctly; expected 3 spaces but found 4
   |       |     (Generic.Arrays.ArrayIndent.CloseBraceIncorrect)
------------------------------------------------------------------------------------------------
PHPCBF CAN FIX THE 2 MARKED SNIFF VIOLATIONS AUTOMATICALLY
------------------------------------------------------------------------------------------------
```
An attempt to auto-fix the file fails:
```
PHPCBF RESULT SUMMARY
----------------------------------------------------------------------
FILE                                                  FIXED  REMAINING
----------------------------------------------------------------------
test.php                                              FAILED TO FIX
----------------------------------------------------------------------
A TOTAL OF 0 ERRORS WERE FIXED IN 1 FILE
----------------------------------------------------------------------
PHPCBF FAILED TO FIX 1 FILE
----------------------------------------------------------------------
```
And introduces another violation (both lines 8 and 9 are now indented using 3 spaces instead of 4):
```php
<?php

declare(strict_types=1);

$foo =
[
    'bar' =>
   [
   ],
];
```
An attempt to fix the modified file also fails. Note that fixing the file using `--standard=Squiz` (as per the reported violations) succeeds:
```sh
$ phpcbf --standard=Squiz test.php

PHPCBF RESULT SUMMARY
----------------------------------------------------------------------
FILE                                                  FIXED  REMAINING
----------------------------------------------------------------------
/home/morozov/Projects/dbal/test.php                  4      1
----------------------------------------------------------------------
A TOTAL OF 4 ERRORS WERE FIXED IN 1 FILE
----------------------------------------------------------------------
```
```php
<?php

declare(strict_types=1);

$foo = [
    'bar' => [],
];
```

**Comment by kukulich** [comment]
@morozov both errors are reported by sniffs from PHPCS itself. There are no errors reported by our sniffs.

**Comment by morozov** [comment]
@kukulich yes, but the issue is not reproducible with any of the OOTB standards (I tried Squiz, PSR-2 and PSR-12). Otherwise, I would have reported it upstream. Additionally, `phpcbf -vv` shows that some `Slevomat` standards taking place during the fix, although I didn't dig deeper.

How would you suggest approaching the cases where it's not clear where exactly the problem is?

**Comment by kukulich** [comment]
@morozov 

The bug is reproducible with this configuration:

```xml
<?xml version="1.0"?>
<ruleset>
    <rule ref="Generic.WhiteSpace.ScopeIndent">
        <properties>
            <property name="ignoreIndentationTokens" type="array">
                <element value="T_COMMENT"/>
                <element value="T_DOC_COMMENT_OPEN_TAG"/>
            </property>
        </properties>
    </rule>
    <rule ref="Generic.Arrays.ArrayIndent"/>
</ruleset>
```

The first sniff is added as a part of [PSR2](https://github.com/doctrine/coding-standard/blob/7.0.2/lib/Doctrine/ruleset.xml#L10), the second is specified directly in [Doctrine CS ruleset.xml](https://github.com/doctrine/coding-standard/blob/7.0.2/lib/Doctrine/ruleset.xml#L20).

PSR-2 and PSR-12 cannot reproduce this error because `Generic.Arrays.ArrayIndent` is not part of these standards.
Squiz cannot report it because it doesn't contain `Generic.WhiteSpace.ScopeIndent`.

So please report in PHPCS :)

**Comment by morozov** [comment]
> So please report it in PHPCS :)

Thank you, @kukulich! I’m sorry for the false-positive but I can’t promise I won’t do it again.
